src/main/java/seedu/address/logic/parser/IssueParserUtil.java

```diff
@@ -0,0 +1,98 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.Deadline;
+import seedu.address.model.issue.Description;
+import seedu.address.model.issue.Priority;
+import seedu.address.model.issue.Status;
+import seedu.address.model.project.Project;
+import seedu.address.model.project.ProjectId;
+import seedu.address.model.project.UniqueProjectList;
+
+/**
+ * Contains utility methods used for parsing strings in the various *Parser classes.
+ */
+public class IssueParserUtil {
```

Comment:
Is it possible to use the overall `ParserUtil` classes? Otherwise parser methods such as `parseProject` may involve code duplication, and also necessitates the change in the overall structure where each entity class requires its own `ParserUtil` class e.g. `ProjectParserUtil`, `ClientParserUtil`